themes/red-theme/lms/templates/header.html

```diff
@@ -165,7 +166,7 @@ <h1 class="logo">
 </header>
 % if course:
 <!--[if lte IE 8]>
-<div class="ie-banner" aria-hidden="true">${_(HTML('<strong>Warning:</strong> Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.')).format(chrome_link='<a href="https://www.google.com/intl/en/chrome/browser/" target="_blank">Chrome</a>', ff_link='<a href="http://www.mozilla.org/en-US/firefox/new/" target="_blank">Firefox</a>')}</div>
+<div class="ie-banner" aria-hidden="true">$(Text(_('{strong_open_tag}Warning:{strong_close_tag} Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.')).format(chrome_link=HTML('<a href="https://www.google.com/intl/en/chrome/browser/" target="_blank">Chrome</a>'), ff_link=HTML('<a href="http://www.mozilla.org/en-US/firefox/new/" target="_blank">Firefox</a>'), strong_open_tag=HTML('<strong>'), strong_close_tag=HTML('</strong>'),)}</div>
```

Comment:
I prefer at least having the format arguments each on their own line like the rest of these.